pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -467,6 +471,34 @@ func (az *Cloud) ensurePublicIPExists(service *v1.Service, pipName string, domai
 	return &pip, nil
 }
 
+func getIdleTimeout(s *v1.Service) (*int32, error) {
+	const (
+		min = 4
+		max = 30
+	)
+	errInvalidTimeout := fmt.Errorf("idle timeout value must be a whole number representing minutes between %d and %d", min, max)
```

Comment:
no need to define a var here, put it under
`return nil, fmt.Errorf("idle timeout value must be a whole number representing minutes between %d and %d", min, max)`

Reply:
I'm using this multiple times.